experimental/virtio/src/vsock/socket/mod.rs

```diff
@@ -16,24 +16,24 @@
 
 use super::{
     packet::{Packet, VSockFlags, VSockOp, HEADER_SIZE},
-    VSock, DATA_BUFFER_SIZE, QUEUE_SIZE,
+    VSock, DATA_BUFFER_SIZE,
 };
 use alloc::collections::VecDeque;
 use core::num::Wrapping;
 use rust_hypervisor_firmware_virtio::virtio::VirtioTransport;
 
 /// The maximum buffer size used by the socket.
 ///
-/// This is used for flow-control calculations. For now we use the maximum size seeing that we don't
-/// have a fixed limit and don't want to send too many credit update packets.
-const STREAM_BUFFER_LENGTH: Wrapping<u32> = Wrapping(u32::MAX);
+/// This is used for flow-control calculations. Seeing that we don't use an actual streambuffer,
+/// this is currently an arbitrary value that seems to avoid stalling.
+const STREAM_BUFFER_LENGTH: Wrapping<u32> = Wrapping(1024 * 1024);
```

Comment:
To rephrase, we currently don't use an actual stream buffer in the guest (since just use the available memory). This number therefore only concerns what we communicate to the host.

Correct?

Reply:
Yes, that is correct.